docs/guides/documentation-style.stories.mdx

```diff
@@ -0,0 +1,87 @@
+consumers.
+
+<br />
+
+## Principles
+
+<br />
+
+### Minimum viable correctness
+
+| **✅ Do**                                                         | **❌ Don't**                                           |
+| :---------------------------------------------------------------- | :----------------------------------------------------- |
+| Provide the minimum amount of information necessary to be correct | Attempt to preempt every possible question or use case |
+
+Less is more. The more we write...
+
+- the more consumers must read
+- the more we must maintain
+
+We can always write more later as-needed to reflect questions that consumers
+bring to the team, or as the system evolves.
+
+<br />
+
+### Document for Jobber
+
+Follow Atlantis content guidelines for written content and component examples:
+
+- [formatting](../?path=/docs/content-formatting--docs)
+- [voice & tone](../?path=/docs/content-voice-tone--docs)
+- [vocabulary](../?path=/docs/content-product-vocabulary--docs)
+
+#### Relevance
+
+| **✅ Do**                              | **❌ Don't**                               |
+| :------------------------------------- | :----------------------------------------- |
+| Use realistic examples and sample data | Insert your favourite fictional characters |
+
+As tempting as it is to use characters from _insert your favourite
+book/movie/show_ into a component example, not everyone consumes the same media
+as you and the reference may not be understood.
+
+<br />
+
+### Show show show... then tell
+
+| **✅ Do**                          | **❌ Don't**                         |
+| :--------------------------------- | :----------------------------------- |
+| <img src={showDo} width="300px" /> | <img src={showDont} width="300px" /> |
+
+Embed `Stories` inline, share working code snippets, and use clear **✅ Do** /
+**❌ Don't** examples to give people what they need with less effort.
+
+<br />
+
+### Use hierarchy
+
+| **✅ Do**                    | **❌ Don't**                  |
+| :--------------------------- | :---------------------------- |
+| **Design guidance**          | **Design guidance**           |
+| -> When to use               | -> When to use                |
+| -> Related components        | **Callbacks**                 |
+| -> Content guidelines        | **Related components**        |
+| -> Accessibility             | -> Accessibility              |
+| **Implementation guidance**  | **Controlled vs uncontroled** |
+| -> Props                     | **Responsiveness**            |
+| -> Callbacks                 | **Component composition**     |
+| -> Component composition     | -> Props                      |
+| -> Controlled vs uncontroled |                               |
+
+Clearly distinguish code-specific guidance from conceptual guidance so consumers
+can find what they're looking for without feeling overwhelmed.
+
+The template generated by `npm run generate` should be your starting point.
```

Comment:
@scotttjob writing this made me realize/wonder... 

a) Does `npm run generate` still work? 
b) if not, is there an updated "boilerplate" command we should be referencing

Reply:
(Update: confirmed it still works)

Reply:
It does work! At least it's still there. I didn't use it myself.